dotnet/samples/GettingStartedWithProcesses/GettingStartedWithProcesses.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is the project using c# >9 only? I suppose so... if the C# required is lower, maybe revert to explicit type declarations...

Reply:
I believe we're using .NET 8.0 which is associated with C# V12

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

The version management for the build system is mamaged globally for the entire repo.

Reply:
It was not visible so I was assuming this can be run with different c# versions

Reply:
We used to target multi versions of dotnet (6,7,8) - most recently we've shifted to only supporting dotnet 8.0...I think there's also a dotnet standard 2.0 configuration.